README.md

```diff
@@ -1,12 +1,50 @@
 # Diffyscan
 
-Diff your Github code against Etherscan verified source code.
+![python ^3.10](https://img.shields.io/badge/python-^3.10-blue)
+![poetry ^1.4](https://img.shields.io/badge/poetry-^1.4-blue)
+![license MIT](https://img.shields.io/badge/license-MIT-brightgreen)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+Diff your Ethereum smart contracts code from GitHub against Etherscan verified source code.
+
+## Prerequisites
+This project was developed using these dependencies with their exact versions listed below:
+- Python ^3.10
+- Poetry ^1.4
+
+Other versions may work as well but were not tested at all.
+
+## Setup
+
+1. Install Poetry
+
+Use the following command to install poetry:
+
+```shell
+pip install --user poetry~=1.4
+```
+
+alternatively, you could proceed with `pipx`:
+
+```shell
+pipx install poetry~=1.4
+```
+
+2. Install Python dependencies
```

Comment:
Not sure why but `poetry install` => `poetry shell` sometimes fails for me in some poetry projects. On the other hand, `poetry shell` first => `poetry install` works every time

Reply:
Np, changed the order :+1: